src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/format/FormatExecutionContext.kt

```diff
@@ -0,0 +1,12 @@
+package org.jmailen.gradle.kotlinter.tasks.format
+
+import org.gradle.api.logging.Logger
+import org.jmailen.gradle.kotlinter.support.ExecutionContext
+
+/**
+ * Execution context for the formatting task.
+ */
+data class FormatExecutionContext(
+    override val logger: Logger,
+    val fixes: MutableList<String> = mutableListOf()
```

Comment:
Does this collection need to be threadsafe?

Reply:
Yes, I realized that after I pushed the changes - you'll get a fix on Monday...